apps/mobile-alphabet/src/app/Components/alphabet-card-detail.screen.tsx

```diff
@@ -95,56 +99,65 @@ export function AlphabetCardDetailScreen({ route }) {
-                    '.mp3',
-                    ''
-                )}`}
-            />
-
-            <AppAudio
-                message="Play Word"
-                url={`${BASE_API_URL}/resources/mediaitems/download?name=${word_audio.replace(
-                    '.mp3',
-                    ''
-                )}`}
-            />
-
-            <Button
-                testID="Back"
-                title="Back"
-                onPress={() => {
-                    setSelectedLetterSequenceNumber(
-                        ((selectedLetterSequenceNumber - 2 + alphabetCards.length) %
-                            alphabetCards.length) +
-                            1
-                    );
-                }}
-            />
-            <Button
-                testID="Next"
-                title="Next"
-                onPress={() => {
-                    setSelectedLetterSequenceNumber(
-                        (selectedLetterSequenceNumber % alphabetCards.length) + 1
-                    );
-                }}
-            />
+            <View style={alphabetDetailStyle.layout}>
+                <View style={alphabetDetailStyle.flexLayout}></View>
+                <View style={alphabetDetailStyle.flexLayout}>
+                    <Text style={alphabetDetailStyle.letter} testID={`${letter}`}>
+                        {letter}
+                    </Text>
+                </View>
+                <View style={alphabetDetailStyle.appAudio}>
+                    <AppAudio
+                        message={letter}
+                        url={`${BASE_API_URL}/resources/mediaitems/download?name=${letter_audio.replace(
+                            '.mp3',
+                            ''
+                        )}`}
+                    />
+                </View>
+            </View>
+            <View style={alphabetDetailStyle.layout}>
+                <View style={alphabetDetailStyle.flexLayout}></View>
+                <View style={alphabetDetailStyle.flexLayout}>
+                    <Text style={alphabetDetailStyle.word} testID={`${word}`}>
+                        {word}
+                    </Text>
+                </View>
+                <View style={alphabetDetailStyle.appAudio}>
+                    <AppAudio
+                        message={word}
+                        url={`${BASE_API_URL}/resources/mediaitems/download?name=${word_audio.replace(
+                            '.mp3',
+                            ''
+                        )}`}
+                    />
+                </View>
+            </View>
+
+            <View style={alphabetDetailStyle.navigationButtons}>
+                <TouchableOpacity
+                    style={{ padding: 20 }}
+                    onPress={() => {
+                        setSelectedLetterSequenceNumber(
+                            ((selectedLetterSequenceNumber - 2 + alphabetCards.length) %
```

Comment:
Would it make sense to create functions for this and the other `setSelectedLetterSequenceNumber()` call like `increment` and `decrement` to make this more readable?